modules/aws-managed-prometheus/locals.tf

```diff
@@ -4,7 +4,7 @@ locals {
   irsa_config = {
     ingest = {
       service_account             = "amp-ingest",
-      create_kubernetes_namespace = true,
+      create_kubernetes_namespace = false,
```

Comment:
Amazon Managed Prometheus module creation is failing due to prometheus namespace not found.

│ Error: namespaces "prometheus" not found
│
│ with module.aws-eks-accelerator-for-terraform.module.aws_managed_prometheus[0].module.irsa["query"].kubernetes_service_account_v1.irsa[0],
│ on /Users/vabonthu/Documents/GITHUB/aws-eks-accelerator-for-terraform/modules/irsa/main.tf line 30, in resource "kubernetes_service_account_v1" "irsa":
│ 30: resource "kubernetes_service_account_v1" "irsa" {
│
╵